### ISSUE TITLE
Query - functionality of "notify" and link with CSV

### ISSUE THREAD

**Issue by turley85** [AtlasOfLivingAustralia/alerts#289]
When creating a new alert, when I click "notify" does this add the related alerts to the CSV?

Or will this only occur after the alert is scheduled?

**Comment by nickdos** [comment]
Next to `Notify` button it says:

> "Notify" will send alerts to the subscribers and update the last check date to today

Which is the same as the regular weekly emails - which also update the the "last check date". So CSV is updated in both cases.

**Comment by turley85** [comment]
Ok, this is a problem. As we basically need to use the "notify" button to schedule an alert on a particular day, this is occurring before we add external users to the alert. As such, the CSV is recording data/alerts that were never sent to the external users, which is not what we want. 

I think this issue has arisen as we've used the notify function to both schedule an alert, and notify users. Happy to talk though this more in our catch up tomorrow.

**Comment by turley85** [comment]
Actually, I'm a bit confused. How do we schedule an alert for a particular date again. The functionality I thought we had isn't actually there...

**Comment by nickdos** [comment]
Yes, needs some thought, as there might be implications of doing it the other way - no update of DB and CSV files. 

It sounds like the `Notify` should work the same as `Preview` but I'm guessing it was done the way it is, for "reasons" (probably technical ones). We might need to wait for Bai to return from leave before committing to a change, just so we don't replay history.

**Comment by nickdos** [comment]
Just adding another snippet of info on the notify function. It seems to ignore the "Check alert since" date picker (at least for existing alerts that have run via the schedule) and uses the last check date as the "previous" date. This makes sense as it is updating the database and will mean you will never double-up entries in the CSV file.

**Comment by qifeng-bai** [comment]
@nickdos @turley85 The 'Notify' button should only be used if the server unexpectedly goes down during a scheduled run or other unexpected failures, requiring the task to be triggered manually. It wil send emails and generate CSV files. Otherwise, this button should not be used.

The 'Preview' button is primarily for administrators to verify that a query runs correctly. It does not update the last execution date, send emails, or regenerate a CSV

**Comment by qifeng-bai** [comment]
@nickdos The use of the date picker depends on how the query is constructed — it’s only used when using Preview. For the Notify button, the start date is taken from the last checked date stored in the database.

This can be confusing, so it would be clearer to group the date picker and Preview button together in a single block.

**Comment by nickdos** [comment]
> @nickdos @turley85 The 'Notify' button should only be used if the server unexpectedly goes down during a scheduled run or other unexpected failures, requiring the task to be triggered manually. It wil send emails and generate CSV files. Otherwise, this button should not be used.

@qifeng-bai - sounds like the button needs some additional help text, to clarify this.

**Comment by qifeng-bai** [comment]
@nickdos @kylie-m  Help added, any suggestions?

<img width="282" height="283" alt="Image" src="https://github.com/user-attachments/assets/cb27ed66-a77a-4035-8f71-afe074c2cc6e" />

**Comment by kylie-m** [comment]
I think that works Bai. If you wanted to do what you suggested earlier by separating the buttons that could help make it even clearer - 
> This can be confusing, so it would be clearer to group the date picker and Preview button together in a single block.


eg two options below, 1 with all info upfront, and 2 with an expanding info section - but this might be a lot of extra work

<img width="1062" height="598" alt="Image" src="https://github.com/user-attachments/assets/ca5218b0-a7b0-40c3-924c-682ba1f5891d" />
